pkg/webhooks/router/admission.go

```diff
@@ -58,6 +58,8 @@ func ForEachAdmission(config *options.Config, handler func(*AdmissionService) er
 			if err := handler(service); err != nil {
 				return err
 			}
+		} else {
+			return fmt.Errorf("enabled admission %s not found on the admission registered map", admission)
```

Comment:
Why is there an error if admission is not found?

Reply:
when i registered a custom admission to the Volcano webhook manager, an issue occurred where the admission was registered incorrectly due to my input error. This problem was very difficult to troubleshoot because it doesn't explicitly tell you that the admission you specified was actually not registered.